### PR TITLE
Display instructions in generated translated files

### PR DESCRIPTION
**Pull request by Tinkidinki**
## Purpose / Description
Explained in the title

## Fixes
Fixes issue #10557

## Approach
As described in the issue, the current header has been *replaced* with the instructions. This is because the translated files already contain the license header, so there doesn't seem to be a need for a repeat. 

I also want to confirm that this is okay, or whether the instructions have to be appended to the license information. 

The current file's header has also been modified in two places, to (1) correct the path of the Ankidroid translations zip file, and (2) a variable name in the instructions. 

Would like to confirm that these two changes are okay as well. 

## How Has This Been Tested?
I was able to make some changes to sync_translations.yml and run the Sync Translations workflow to ensure changes are made. 

## Learning (optional, can help others)
I found the blogpost on Github Workflow syntax to be useful: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ ] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: tools/update-localizations.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def replacechars(filename, fileExt, isCrowdin):
...
+                line = "<?xml version=\"1.0\" encoding=\"utf-8\"?> \n <!-- \n ~ THIS IS AN AUTOMATICALLY GENERATED FILE. PLEASE DO NOT EDIT THIS FILE. \n ~ 1. If you would like to edit the original xml file, find it at: Anki-Android/AnkiDroid/src/main/res/values \n ~ 2. If you would like to provide a translation of the original file, you may do so using Crowdin. \n ~    Instructions for this are available here: https://github.com/ankidroid/Anki-Android/wiki/Translating-AnkiDroid. \n ~    You may also find the documentation on Contributing to Anki useful: https://github.com/ankidroid/Anki-Android/wiki/Contributing \n --> \n \n"
```
Comment (david-allison) [inline tools/update-localizations.py:63]: We shouldn't remove people's copyrights.

When people are here, there are typically two goals:
* Fixing a translation (these instructions are excellent)
* Modify, add or delete strings

For this, we want to guide them to https://github.com/ankidroid/Anki-Android/wiki/Development-Guide#adding-translations which should explain exactly what's needed

**Review by david-allison — CHANGES_REQUESTED** [review]
Needs a few small changes. Looks good!

Could you copy the user-readable message that's displayed, as that's easier to review than a long line with `\n`s in it.

**Comment by Tinkidinki** [comment]
This is the comment that I will be adding:

**Comment by Tinkidinki** [comment]
Is there something else that needs to be done here?

**Review by mikehardy — APPROVED** [review]
This looks good to me, thank you!

File: tools/update-localizations.py [hunk 2]
```diff
@@ -60,7 +60,7 @@ def replacechars(filename, fileExt, isCrowdin):
...
...neral Public License as published by the Free Software \n ~ Foundation; either version 3 of the License, or (at your option) any later \n ~ version. \n ~ \n ~ This program is distributed in the hope that it will be useful, but WITHOUT ANY \n ~ WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A \n ~ PARTICULAR PURPOSE. See the GNU General Public License for more details. \n ~ \n ~ You should have received a copy of the GNU General Public License along with \n ~ this program.  If not, see <http://www.gnu.org/licenses/>. \n ~ \n ~ THIS IS AN AUTOMATICALLY GENERATED FILE. PLEASE DO NOT EDIT THIS FILE. \n ~ 1. If you would like to add/delete/modify the original translatable strings, follow instructions here:  https://github.com/ankidroid/Anki-Android/wiki/Development-Guide#adding-translations  \n ~ 2. If you would like to provide a translation of the original file, you may do so using Crowdin. \n ~    Instructions for this are available here: https://github.com/ankidroid/Anki-Android/wiki/Translating-AnkiDroid. \n ~    You may also find the documentation on contributing to Anki useful: https://github.com/ankidroid/Anki-Android/wiki/Contributing \n --> \n \n"
```
Comment (david-allison) [inline tools/update-localizations.py:63]: I think we should put the "autogenerated file"/instructions before the copyrights in this case. 

We need to ensure that the maximal number of users see this.

Message looks great!

**Review by david-allison — CHANGES_REQUESTED** [review]

File: tools/update-localizations.py (hunk 2 above)
Comment (mikehardy) [reply tools/update-localizations.py:63]: That's a great point. Put it right at the top - that's better. Sorry for asking for another change but I think David's suggestion will really improve it @Tinkidinki - this ends up wasting a lot of people's time, and I hate to see that happen if we can avoid it...

**Comment by Tinkidinki** [comment]
Made the change! I apologise for the delay!

**Comment by Tinkidinki** [comment]
Made a change with the comment symbol

**Review by mikehardy — APPROVED** [review]
David's comment is addressed! Thanks for your patience @Tinkidinki this looks great

**Comment by mikehardy** [comment]
I'm running our translations sync job right now in order to exercise the code :crossed_fingers: :four_leaf_clover: :smile: 

https://github.com/ankidroid/Anki-Android/runs/5872757979?check_suite_focus=true

**Comment by mikehardy** [comment]
Hi there! APRIL OpenCollective Notice

Just a friendly notice that we try to process OpenCollective payments monthly - it's time for [April 2022 submissions](https://github.com/ankidroid/Anki-Android/pulls?q=is%3Apr+merged%3A2022-04-01..2022-04-30)

If you are interested in compensation for this work, the process with details is here:

https://github.com/ankidroid/Anki-Android/wiki/OpenCollective-Payment-Process#how-to-get-paid

I only post one comment per person to avoid spamming you, regardless of the number of PRs merged, but this note applies to all PRs merged for in the month of March.

Please note that GSoC contributions are okay for this process. Our philosophy is that our users have donated to AnkiDroid for all contributions. The only PRs that will not go through the OpenCollective process are ones directly related to am accepted GSoC project from a selected participant, since those receive a stipend from GSoC itself.

Please understand that this means the monthly budget we have for payments might be stretched a bit and the cap on payments-per-person may be lower, but we try to make our process as fair and transparent as possible, we just need your understanding.

Thanks!

**Pull request merged**